content/content-tool/tool/src/java/org/sakaiproject/content/tool/SiteEmailNotificationDragAndDrop.java

```diff
@@ -525,37 +576,40 @@ private void addMessageTextResources(StringBuilder buf, String title, Reference
 		}
 		buf.append(blankLine);
 
-		for (int i=0;i<this.fileList.size();i++)
+		if (this.fileList != null)
```

Comment:
Below changes are merely indentation modifications due to adding `null` check before the loop executes.